### ISSUE TITLE
Does not work with Uglify

### ISSUE THREAD

**Issue by kodeine** [finnfiddle/words-to-numbers#31]
as the title says, i tried using it with `ionic framework` and it compiles with uglify where it doesnt work!

**Comment by krantisapient** [comment]
Yes, even I am facing same issue when I import this plugin to Angular 8 project. I am getting following error.

clj-fuzzy.js:2 Uncaught TypeError: Cannot use 'in' operator to search for 'clj_fuzzy' in undefined
    at ea (clj-fuzzy.js:2)
    at clj-fuzzy.js:5
    at Object../node_modules/clj-fuzzy/src-js/clj-fuzzy.js (clj-fuzzy.js:6)
    at __webpack_require__ (bootstrap:79)
    at Object../node_modules/words-to-numbers/dist/index.js (index.js:21)
    at __webpack_require__ (bootstrap:79)
    at Object../node_modules/words-to-numbers/index.js (index.js:1)
    at __webpack_require__ (bootstrap:79)
